cpp/arcticdb/stream/aggregator.hpp

```diff
@@ -131,12 +131,12 @@ class ListOfSegmentPolicies {
 
 class DenseColumnPolicy {
 public:
-    static constexpr bool allow_sparse = false;
+    static constexpr Sparsity allow_sparse = Sparsity::NOT_PERMITTED;
 };
 
 class SparseColumnPolicy {
 public:
-    static constexpr bool allow_sparse = true;
+    static constexpr Sparsity allow_sparse = Sparsity::PERMITTED;
```

Comment:
What do these classes now achieve that templating directly on `Sparsity` would not?